ValgrindCI/__init__.py

```diff
@@ -55,6 +78,27 @@ def main():
     data.parse(args.xml_file)
     data.set_source_dir(args.source_dir)
 
+    if args.substitute_path:
+        substitute_paths = []
+        for s in args.substitute_path:
+            substitute_paths.append({"from": s.split(":")[0], "to": s.split(":")[1] })
+        data.set_substitute_paths(substitute_paths)
```

Comment:
`s.split(':')` is executed twice which is inefficient. Please use local variables instead:
```suggestion
        for s in args.substitute_path:
            _from, _to = s.split(':')
            substitute_paths.append({"from": _from, "to": _to})
        data.set_substitute_paths(substitute_paths)
```